Khan/perseus
# [Locked Figure Labels] Add/edit/delete locked function labels

## Summary:
Add the ability to add/edit/delete labels for locked functions
within the interactive graph editor.

Issue: https://khanacademy.atlassian.net/browse/LEMS-2351

## Test plan:
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-function-labels-flag
- Go to locked figures
- Open the locked function settings
- Confirm that the label settings are there
- Play around with the label settings and confirm that
  the color updates with the function.

<img width="354" alt="image" src="https://github.com/user-attachments/assets/874f950c-b19d-4f54-a3e1-22da680fb172">
